renku/service/controllers/datasets_edit.py

```diff
@@ -45,8 +45,16 @@ def context(self):
         """Controller operation context."""
         return self.ctx
 
+    def _handle_uploaded_images(self):
```

Comment:
should we move this method to `renku.service.utils` somewhere and call it like `handle_uploaded_images(self.ctx.get("images", []), self.cache, self.user)` in the controls, so there's no duplication?

It not a lot of lines of code, but still a bit cleaner